### PR TITLE
chore: add tests for toPlainObject

### PR DESCRIPTION
**Pull request by marcolink**
I was a bit confused by the naming of the function, so I thought it makes sense to cover it with some tests to make it clear what it does.

### CODE REVIEW

**Review by matthew-contentful — APPROVED** [review]
Einfach, ordentlich, schnell :)

File: test/unit/to-plain-object-test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        }
+    }
+
+    const obj = new TestClassObject('class object');
+    const result = toPlainObject(obj);
+
+    t.equals(obj instanceof TestClassObject, true, 'obj is instanceof TestClassObject');
+    t.equals(obj, result, 'toPlainObject returns same object');
+    t.equals(typeof result.toPlainObject === 'function', true, 'enhanced object has function "toPlainObject"');
+    t.notEqual(result, result.toPlainObject(), 'obj.toPlainObject() returns copy');
```
Comment (TimBeyer) [inline test/unit/to-plain-object-test.js:17]: Should we also add a test to see what it actually is equal to?

Comment (marcolink) [reply test/unit/to-plain-object-test.js]: @TimBeyer  Good point, I added more tests.

Comment (TimBeyer) [reply test/unit/to-plain-object-test.js]: Actually the method doc string says 
> Mixes in a method to return just a plain object with no additional methods
But then it looks like it does copy everything including the methods.
Maybe we should check if it does what it promises, though I'm not sure how to proceed if it indeed does copy all the things.

**Review by suevalov — APPROVED** [review]

**Review by suevalov — CHANGES_REQUESTED** [review]
Could you please rebase and rewrite test to Jest?

**Comment by marcolink** [comment]
@suevalov check!

**Pull request merged**